### ISSUE TITLE
Project template doesn´t work

### ISSUE THREAD

**Issue by SNDK77** [salesagility/SuiteCRM#1275]
I create a project template and when I create a new project in Project Module and select the template doesn´t create the associated tasks.

If I go to Project Templates Module and open the template and with the upper button create a new project then works fine and create the associated tasks founds in template.

Suitecrm Versión 7.5.3 Sugar Versión 6.5.23 (Build 1061)
Tipo de servidor: MySQL
Versión del servidor: 5.6.28 - MySQL Community Server (GPL)
Chrome 49.0.2623.112 m

**Comment by recci** [comment]
This is not a bug, you dont select the project template from inside the project module and expect it to create the Tasks. You create a new project from a Template from within the project templates module thats how its meant to work.

**Comment by 2xaronl** [comment]
Where is this button located to create the project from a template?  I don't see it in 7.7 with the SuiteR theme.

I, too, initially wanted to create a project using a template the same way SNDK77 did.  It seems more intuitive.  Please implement this.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Project template doesn´t work](https://trello.com/c/GbuwUoHJ/105-project-template-doesn-t-work)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
